config/crd/bases/pingcap.com_iochaos.yaml

```diff
@@ -149,6 +149,13 @@ spec:
                   items:
                     type: string
                   type: array
+                phaseSelector:
```

Comment:
it's better to  replace with `phaseSelectors`?

Reply:
This is auto-generated by executing `make manifests`